test/test_listen_handler.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Jeny Sadadia
+# Author: Jeny Sadadia <jeny.sadadia@gmail.com>
```

Comment:
@gctucker Should Alex be mentioned here too? She is the author of the test cases.

Reply:
@JenySadadia Please, add another author header with Alex's name and e-mail `
Alexandra Pereira` `alexandra.pereira@collabora.com`

Reply:
Yes, in fact @alexandrasp wrote the code that was copied here so she's the only one required to be listed.  It's fine to also have Jeny, I think both should be listed.  The copyright for Alex's changes should be Collabora, like in other files.